node/src/jasmine_retry.ts

```diff
@@ -54,5 +54,7 @@ export function retryFailedTests(retries: number, millisecondsBetweenRetries: nu
     }
     return spec
   }
+  // returns the original message that Jasmine uses to identify pending specs
+  typelessJasmine.Spec.pendingSpecExceptionMessage = '=> marked Pending'
```

Comment:
Is it supposed to solve the problem of pending tests being marked as failed? If yes, please add a pending test to the example project to make sure it works.